### ISSUE TITLE
Keyboard navigation using arrow keys doesn't work in several places 

### ISSUE THREAD

**Issue by alexjlockwood** [alexjlockwood/ShapeShifter#133]
For example, if you select a path and try to navigate through the tree of paths using the up/down arrow keys, nothing happens.

On an unrelated note, I would guess that when one of the layer items has focus, there would be a keyboard shortcut to potentially expand/collapse that layer if possible.
